e2e/helpers/generated_order_helper.js

```diff
@@ -196,6 +202,8 @@ module.exports = {
       I.seeInTab([orderHeading, 'Order title'], order.title);
       I.seeInTab([orderHeading, 'Order document'], order.document);
       I.seeInTab([orderHeading, 'Date on order'], dateFormat(defaultIssuedDate, 'd mmmm yyyy'));
+    } else if (order.type === 'Emergency protection order') {
+      I.seeTextInTab([orderHeading, 'Date on order']);
```

Comment:
previously document has been verified, and is verified for other orders

Reply:
beside there are 
Date on order: 22 October 2020, 12:00am
Date and time of upload: 9:07am, 22 October 2020

different date format, please confirm with UX

Reply:
Checking with UX... date format was my mistake, but now they are questioning why we show date of upload - they want to see end date too

Reply:
we will go with time then date (TIME_DATE). Also labels have been updated+ reordered: "Starts on" "Ends on"